src/main/java/net/fuzzycraft/botanichorizons/patches/ManaInfusionPatches.java

```diff
@@ -186,7 +194,32 @@ public static void applyPatches() {
         ModManaAlchemyRecipes.flowersRecipes.add(BotaniaAPI.registerManaAlchemyRecipe(new ItemStack(Blocks.double_plant, 1, 4), new ItemStack(Blocks.double_plant, 1, 1), 400));
         ModManaAlchemyRecipes.flowersRecipes.add(BotaniaAPI.registerManaAlchemyRecipe(new ItemStack(Blocks.double_plant, 1, 5), new ItemStack(Blocks.double_plant, 1, 4), 400));
         ModManaAlchemyRecipes.flowersRecipes.add(BotaniaAPI.registerManaAlchemyRecipe(new ItemStack(Blocks.yellow_flower), new ItemStack(Blocks.double_plant, 1, 5), 400));
-
+            if(Loader.isModLoaded("BiomesOPlenty")){
+        ModManaAlchemyRecipes.flowersRecipes.add(BotaniaAPI.registerManaAlchemyRecipe(OreDictionary.getOres("flowerClover").get(0),"flowerSwamp", 8000));
```

Comment:
Indentation error

Reply:
Don't resolve unfixed issues.

Reply:
I thought I changed this file yesterday wtf

Reply:
Yes, is fixed in the last one

Reply:
You unindented the first line, but you still haven't indented everything within {...}